src/docs/product/alerts-notifications/index.mdx

```diff
@@ -6,26 +6,40 @@ redirect_from:
 description: "Sentry categorizes notifications into alerts and other types of notifications. Learn more about how these are managed."
 ---
 
-Sentry categorizes notifications broadly into alerts and non-alert notifications.
+Alerts notify you about errors and performance problems in your application. Notifications send emails for situations like an issue state change or if a release has deployed. Alerts and notifications are customizable to your organization's needs with options to pair with [integrations](/product/integrations/), flexible configurations, and choices for both error or performance alerting. Default settings for notifications are also available.
 
-## Alerts
+## Issue Alerts for Errors
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
+Issue alerting sets alerts on changes within grouped error events, or _issues_. Issues are grouped automatically by Sentry based on stack trace and other factors. Create alerts for issue-level changes, such as:
```

Comment:
why did we change this? "Sentry automatically groups individual error events into Issues based on stack trace and other factors." seems clearer than "Issues are grouped automatically by Sentry based on stack trace and other factors.". Issues are not grouped, error events are